Anlab.Mvc/Client/components/specs/Summary.spec.tsx

```diff
@@ -21,7 +21,8 @@ describe('<Summary />', () => {
                                                 hideError={true}
                                                 isCreate={true}
                                                 onSubmit={null}
-                                                testItems={testItems} />);
```

Comment:
@jSylvestre I wasn't sure how to properly adjust the spec for this so I just added processingFee as null for this. Is that okay?

Reply:
@laholstege Yes, that is fine. That is what I would have done.